addons/account_payment/views/account_portal_templates.xml

```diff
@@ -32,15 +90,31 @@
     </template>
 
     <template id="portal_invoice_payment" name="Invoice Payment">
-        <div class="row">
+        <div id="o_portal_invoice_payment" class="row">
             <div class="modal fade" id="pay_with" role="dialog">
                 <div class="modal-dialog">
                     <div class="modal-content">
                         <div class="modal-header">
-                            <h3 class="modal-title">Pay with</h3>
+                            <h3 class="modal-title">Pay</h3>
                             <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
                         </div>
                         <div class="modal-body">
+                            <div class="text-bg-light row row-cols-1 row-cols-md-2 mx-0 py-2 rounded">
```

Comment:
Seen that code two times, maybe we could do a template to call to factorize that a bit